frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -77,9 +82,84 @@ pods:
         goal: FINISHED
         cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool cleanup -p {{TASKCFG_ALL_JMX_PORT}} -- $CASSANDRA_KEYSPACE_TABLES
         resource-set: sidecar-resources
+      backup-schema:
+        goal: FINISHED
+        cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/cqlsh --cqlversion="3.4.0" -e "DESC SCHEMA" node-0-server.{{SERVICE_NAME}}.mesos > container-path/schema.cql
```

Comment:
Can the `--cqlversion` here and elsewhere be removed or is it needed for this to work? (I don't know either way, just bringing it up)